src/UI/Internal/SideBar.elm

```diff
@@ -189,13 +189,15 @@ headerView cfg toggleMsg logo =
 
 slimHeaderView : RenderConfig -> msg -> Maybe (Menu.Logo msg) -> Element msg
 slimHeaderView cfg toggleMsg _ =
-    Element.column [ height (px (72 + 48)) ]
-        [ (cfg |> localeTerms >> .sidebar >> .expand)
-            |> Icon.sandwichMenu
-            |> Icon.withSize Size.small
-            |> Icon.renderElement cfg
-            |> Element.el (headerButtonAttr toggleMsg)
-        ]
+    (cfg |> localeTerms >> .sidebar >> .expand)
+        |> Icon.sandwichMenu
+        |> Icon.renderElement cfg
+        |> Element.el
+            (headerButtonAttr toggleMsg
+                ++ [ Element.centerX
+                   , Element.paddingEach { top = 8, left = 8, right = 8, bottom = 88 }
```

Comment:
I was checking the here and it has 2 levels of padding, because the has extra padding.

![Screen Shot 2020-10-09 at 17 40 49](https://user-images.githubusercontent.com/1559013/95603527-b1b7a580-0a56-11eb-99e2-948a3422cd07.png)

As we are trying to do the 8dot grid, I would set the top field to 16px